actions/add-review-url/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "AddReviewUrl",
+  "version": "1.0.0",
+  "description": "",
+  "main": "AddReviewUrl.js",
+  "scripts": {
+    "build": "babel src -d lib",
+    "buildtest": "babel test -d spec",
+    "test": "jasmine spec/addReviewUrlTest.js"
+  },
+  "keywords": [],
+  "author": "s-tuli",

```

Comment:
Probably don't want to use your name on this, we'll need to figure out what to put here. What do other actions use?

Reply:
yes. Updated.